### ISSUE TITLE
Tests failing

### ISSUE THREAD

**Issue by mjhouse** [mkhan45/math_lang#21]
main.rs/tests::test_basic fails at this assertion

```rust
        assert_eq!(
            full_eval("5 kilograms + 4 grams").to_string(),
            "5004 g".to_string()
        );
```

With the following message:
```bash
thread 'tests::test_basic' panicked at 'assertion failed: `(left == right)`
  left: `"9000 g"`,
 right: `"5004 g"`', src/main.rs:57:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by mkhan45** [comment]
I'm aware of this; it will be fixed by #20.

**Comment by mjhouse** [comment]
@mkhan45 Alright.

**Comment by mkhan45** [comment]
This specific test and hopefully all similar ones are fixed by 564e317
